image.h

```diff
@@ -20,18 +20,35 @@ class Image {
 	uint16_t pixelIndex(uint16_t x, uint16_t y) {
```

Comment:
These functions themselves can be made `const`, so that they are callable by const references as well

Reply:
True. Done